deeppavlov/core/trainers/fit_trainer.py

```diff
@@ -165,7 +165,7 @@ def get_chainer(self) -> Chainer:
     def train(self, iterator: Union[DataFittingIterator, DataLearningIterator]) -> None:
         """Call :meth:`~fit_chainer` with provided data iterator as an argument"""
         self.fit_chainer(iterator)
-        self._saved = True
+        self._saved = True # FIXME: Should we really set this flag without actually saving the model?
```

Comment:
every fitted component is saved in the `self.fit_chainer(iterator)` method

Reply:
OK